packages/contracts/src/Game.sol

```diff
@@ -1032,4 +1032,12 @@ contract Game {
     }
 
     // ---------------------------------------------------------------------------------------------
+
+    // Returns true if a `player` is currently in an active game
+    // This works because players cannot participate in multiple games simultaneously
+    function playerActive(address player) external view returns(bool){
```

Comment:
This misses a space, you can run `make lint` in the contracts dir to lint (and `make check` to run other checks).